st2actions/st2actions/runners/__init__.py

```diff
@@ -136,9 +138,13 @@ def _get_common_action_env_variables(self):
         result['ST2_ACTION_PACK_NAME'] = self.get_pack_name()
```

Comment:
Let's please move this to a different place.

As I noted here https://github.com/StackStorm/st2/issues/2693#issuecomment-220262497, there are two different things:

1. Environment variables which are set for st2client to work (as mentioned, that's a hack, I don't like it but that's limitation of st2client right now)
2. Environment variables which are set as a convenience and available to actions.

In this case we are fixing 1), but you are doing in in code for 2).